docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -18,6 +18,17 @@ New Features
 * The default Apache configuration has changed in order to support deferred content
   downloading. All requests to ``/pulp/repos`` are now re-written to ``/pulp/content``.
 
+* Yum metadata files and DRPMs can now be copied from one repository to another using CLI.
```

Comment:
No need to document the whole command. I was thinking just a name so it is easy to search for. I am not aware of the new commands though, so I leave it to your judgment.